mobi/src/main/java/com/mobiperf/MeasurementScheduler.java

```diff
@@ -258,20 +263,38 @@ private Notification createServiceRunningNotification() {
         PendingIntent.getActivity(this, 0, intent,
             PendingIntent.FLAG_CANCEL_CURRENT);
 
-    // This constructor is deprecated in 3.x. But most phones still run 2.x systems
-    Notification notice =
-        new Notification(R.drawable.icon_statusbar,
-            getString(R.string.notificationSchedulerStarted),
-            System.currentTimeMillis());
+    Notification notice = new NotificationCompat.Builder(this, CHANNEL_ID)
+            .setSmallIcon(R.drawable.icon_statusbar)
+            .setContentTitle(getString(R.string.notificationSchedulerStarted))
+            .setWhen(System.currentTimeMillis())
+            .setPriority(NotificationCompat.PRIORITY_DEFAULT)
+            .setContentIntent(pendIntent)
+            .build();
     notice.flags |=
-        Notification.FLAG_NO_CLEAR | Notification.FLAG_ONGOING_EVENT;
+            Notification.FLAG_NO_CLEAR | Notification.FLAG_ONGOING_EVENT;
 
     // This is deprecated in 3.x. But most phones still run 2.x systems
    // notice.setLatestEventInfo(this, getString(R.string.app_name),
      //   getString(R.string.notificationServiceRunning), pendIntent);
     return notice;
   }
 
+  private void createNotificationChannel() {
+    // Create the NotificationChannel, but only on API 26+ because
+    // the NotificationChannel class is new and not in the support library
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
if below 26 which channels do the notifications get pushed to

Reply:
Grouping Notification by Chanel ids and Chanel group ids is a feature that is on other APIs